test/source/mock/lib/oauth.ts

```diff
@@ -8,6 +8,8 @@ import { Str } from '../../core/common';
 const authURL = 'https://localhost:8001';
 
 export class OauthMock {
+  public static customIDPClientId = 'ZFfdspDb9Oz1vnFdA4NrCyNaA9N3jYDl';
```

Comment:
could this value be something like "custom-test-idp-client-id" ? Or is it using an actual external idp?

Reply:
It's actual test external idp(my auth0 test idp).
It can also be `custom-test-idp-client-id` but I think setting actual external idp won't be secret issue?

Reply:
Since we have Auth0 too, could you work with Roma to have similar one set up on our Auth0 account?

In that case could name the new one `custom-idp-client-id-for-automated-testing`

I don't see any security issues here at all, the IdP doesn't give access to any actual resources. As long as it's properly named as a test in the Auth0 settings as well.

Reply:
Okay, Got it